tools/test/examples/examples_lib.py

```diff
@@ -30,8 +32,16 @@
 
  """
 
+
+EXECUTABLE_ANALYSIS_TOOLS_PATH = join(
+    os.path.dirname(__file__),
+    "../../executable_analysis_tools/",
```

Comment:
This is require a relative path of the `examples.py` and `executable_analysis_tools` this is not idea, as the `tools\test\examples` folder suppose to be self contained. Is this `executable_analysis_tools` has it standardized CLI ?

Reply:
[The file exported has been moved to the same directory](https://github.com/ARMmbed/mbed-os/compare/51d7e2a046ea2bf7879446185087e3a87bda1584..0ece0cd7e5bd6df966b0ad10e27af1f71c058a82) and therefore no longer requires to be made visible with `sys.path`.